components/scream/scripts/tests/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+cmake_minimum_required(VERSION 3.3)
+cmake_policy(SET CMP0057 NEW)
+
+project(SCRIPTS_TESTS)
+
+enable_testing()
+include(CTest)
+
+add_test(NAME scripts_tests_tas COMMAND ${CMAKE_CURRENT_SOURCE_DIR}/../gather-all-data -l -m $ENV{CIME_MACHINE} "${CMAKE_CURRENT_SOURCE_DIR}/../scripts-tests -f -m \$machine TestTestAllScream")
```

Comment:
Why are there two tests? Are they always both run?

Reply:
One has to be run with gather-all-data to ensure a compute node, the other has to be on a login node.

Reply:
And we need to run both? Shouldn't one run either or, depending on whether it's already on a compute node?

Reply:
Clarified by Jim offline: we need both.